sdk/deploy.js

```diff
@@ -74,8 +74,10 @@ const deployConf = async function deployConf(accountId, funcName, apiKey, funcCo
       }
       if (env[key] == null) {
         env[key] = process.env[key];
-        if (!env[key] || env[key] === '') {
+        if (env[key] === undefined) {
           logger.warn(`Ignoring empty env ${key}`);
+        }
+        if (!env[key] || env[key] === '') {
```

Comment:
Nit: `''` is falsy, so `if (!env[key])` is enough.

But also I'm missing something: this again prevents me from sending an empty environment variable.  (The test below seems to say the same.)